hack/install-requirements.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
I thought about splitting this up to what's needed by g/g --> `hack/install-requirements.sh` and extensions --> `extensions/hack/install-requirements.sh`. What's your opinion?

Reply:
What exactly do you want to split up? This script installs `golangci-lint`, `helm`, and suggests the GNU utils. Everything fits to both, or?

Reply:
The `helm` binary is only required for the extensions [here](https://github.com/gardener/gardener/blob/549d24367f32790a3fcbe980a89c1bc453fe370d/extensions/hack/generate-controller-registration.sh#L69-L70), isn't it?
I isn't meant to be petty, rather a proposal to better separate what is relevant for extensions and what for g/g.

Reply:
Now that we have the dedicated `hack/check-charts.sh` script the extension will call it as well as part of their `make check` rules, and here `helm template` is executed. As the extensions also heavily use Helm charts I'd keep it in this file. All the three requirements make sense to always install when working with g/g + extensions. The specifics are then put into the various `make install-requirements` rules directly. OK?

Reply:
Okay, got it. Thanks for clarifying.

Reply:
👍 Thanks!